sql/database.sql

```diff
@@ -13314,3 +13314,10 @@ CREATE TABLE `patient_settings` (
      `setting_value` varchar(255) NOT NULL DEFAULT '',
      PRIMARY KEY (`setting_patient`, `setting_label`)
 ) ENGINE=InnoDB;
+
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`) VALUES ('lists', 'default_open_tabs', 'Default Open Tabs');
+INSERT INTO `list_options` (`list_id`, `notes`, `title`, `seq`, `option_id`, `activity`) VALUES ('default_open_tabs', 'interface/main_info.php', 'Calendar', 10, 'cal', '1');
```

Comment:
@robertdown , I think there is a flaw here if we assume that the target/option_id will be unique. If peruse the main menu json, there are lots of duplicates in target to ensure they populate the same frame (sometimes necessary to ensure never open up more than 1 patient at a time and other times necessary to avoid opening up a huge number of tabs).

Reply:
actually @robertdown , i wouldn't worry about this issue now. It is something can address in future if ever run into this issue in the future.

Reply:
A lot of tabs require a patient be selected, which wouldn't happen by default.